followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,122 @@
+CallForTenders:
+  label: "Call for tenders"
+  plural: "Calls for tenders"
+  description: >
+    A public appeal issued by an authority, possibly on behalf of another,
+    for buying a specific work, supply or service
+  extends:
+    - Thing
+    - Interval
+  matchable: false
+  required:
+    - title
+    - authority
+  featured:
+    - title
+    - authority
+  caption:
+    - title
+  properties:
+    callId:
+      label: "CfT unique id"
+      type: identifier
+    # What is "CA"? Contract award? Maybe it should then live there...
+    # cftCaUid:
+    #   label: "CfT CA unique id"
+    #   type: identifier
+    title:
+      label: "Title"
+    authority:
+      label: "Name of contracting authority"
+      reverse:
+        name: callForTenders
+        label: "Call For Tenders"
+      type: entity
+      range: LegalEntity
+    onBehalfOf:
+      label: "Published on behalf of"
+      reverse:
+        name: delegatedCallForTenders
+        label: "Delegated call for tenders"
+      type: entity
+      range: LegalEntity
+    publicationDate:
+      label: "Date of publication/invitation"
+      type: date
+    evaluationMechanism:
+      label: "Evaluation mechanism"
+    procurementType:
+      label: "Procurement type"
+    directive:
+      label: "Directive"
+    procedure:
+      label: "Procedure"
+    # What does this mean?
+    # involves:
+    #   label: "Call for tenders involves"
+    cpvCode:
+      label: "CPV codes"
+      description: "Common Procurement Vocabulary (CPV)"
+      type: identifier
+    # eAuctionsIncluded:
+    #   label: "Inclusion of e-Auctions"
+    nutsCode:
+      label: "NUTS codes"
+      description: "Nomenclature of Territorial Units for Statistics (NUTS)"
+      type: identifier
+    relationToThreshold:
+      label: "Above or below threshold"
+    paymentOptions:
+      label: "Payment options"
+    submissionDeadline:
+      label: "Submission deadline"
+      type: date
+    clarificationDeadline:
+      label: "End of clarification period"
+      type: date
+    # clarificationDocumentUpload:
+    #   label: "Upload of documents within the clarification"
+    # notificationsAllowed:
+    #   label: "Allow suppliers to register for notifications"
+    awardedInLots:
+      label: "Contract awarded in Lots"
+    numberOfLots:
+      label: "Number of lots"
+      type: number
+    lotsNames:
+      label: "Lots names"
+    tendersForLots:
+      # What does this mean?
+      label: "Tenders for lots"
+    maximumNumberOfLots:
+      label: "Maximum number of lots"
+      type: number
+    euFunding:
+      label: "EU funding"
+    multipleTenders:
+      label: "Multiple tenders will be accepted"
+    # tedLinksForNotices:
+    #   label: "TED links for published notices"
+    #   -> use sourceUrl instead
```

Comment:
I think this is different to sourceUrl which would be the URL on etenders.gov.mt. TED is the EU tenders journal and I'm not sure every tender that's on etenders (or any other national procurement db) is also automatically on TED. Having this separate field could perhaps allow cross reference to the [TED dataset that's already on Aleph](https://aleph.occrp.org/datasets/755).